### ISSUE TITLE
paddle.unique 在 axis=None，return_inverse = True时，返回的 inverse  shape 与 numpy 和 torch 不一致

### ISSUE THREAD

**Issue by ooooo-create** [PaddlePaddle/Paddle#72822]
### 请提出你的问题 Please ask your question

paddle.unique 在 axis=None，return_inverse = True时，返回的 inverse 是被 flatten 之后的，但是 numpy 和 torch 会保持原来输入的形状
```python
import paddle
import torch
import numpy as np

# 构造一个二维张量
x_np = np.array([[1, 2, 1], [3, 2, 1]])
x_pd = paddle.to_tensor(x_np)
x_th = torch.tensor(x_np)

# paddle.unique
out_pd, inverse_pd = paddle.unique(x_pd, return_inverse=True)
print("Paddle unique output:", out_pd.numpy())
print("Paddle inverse:", inverse_pd.numpy(), "shape:", inverse_pd.shape)

# torch.unique
out_th, inverse_th = torch.unique(x_th, return_inverse=True)
print("Torch unique output:", out_th.numpy())
print("Torch inverse:", inverse_th.numpy(), "shape:", inverse_th.shape)

# numpy.unique
out_np, inverse_np = np.unique(x_np, return_inverse=True)
print("Numpy unique output:", out_np)
print("Numpy inverse:", inverse_np, "shape:", inverse_np.shape)

"""
Paddle unique output: [1 2 3]
Paddle inverse: [0 1 0 2 1 0] shape: [6]
Torch unique output: [1 2 3]
Torch inverse: [[0 1 0]
 [2 1 0]] shape: torch.Size([2, 3])
Numpy unique output: [1 2 3]
Numpy inverse: [[0 1 0]
 [2 1 0]] shape: (2, 3)
"""

```
而且 torch 和 paddle [映射文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.unique.html)中也没有表达出这种差异性
这种差异性是否需要消除或者在文档中说明一下原因~

**Comment by wanghuancoder** [comment]
感谢您的问题反馈，我们尽快修复！
